kong/cli/services/serf.lua

```diff
@@ -28,7 +28,11 @@ function Serf:_get_cmd()
   local cmd, err = Serf.super._get_cmd(self, {}, function(path)
     local res, code = IO.os_execute(path.." version")
     if code == 0 then
-      return res:match("^Serf v0.7.0")
+      local version_match = res:match("^Serf v0.7.0")
+      if not version_match then
+        logger:warn("Incompatible Serf version in the path. Required version is Serf v0.7.0")
```

Comment:
Could it be more concise and to the point, like:

```
Incompatible Serf version. Kong requires v0.7.0.
```

I am not sure mentioning the `$PATH` is worthy?